crates/schema-impl/src/key_extraction.rs

```diff
@@ -602,258 +604,258 @@ pub(crate) mod extract_impls {
-        );
+        // extract_tests!(
+        //     bool,
+        //     item: true,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| encode_varint(if val { 1u64 } else { 0u64 }, buf)
+        // );
+        // extract_tests!(
+        //     int32,
+        //     item: -21314_i32,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| encode_varint(val as u64, buf)
+        // );
+        // extract_tests!(
+        //     int64,
+        //     item: -245361314_i64,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| encode_varint(val as u64, buf)
+        // );
+        // extract_tests!(
+        //     uint32,
+        //     item: 21314_u32,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| encode_varint(val as u64, buf)
+        // );
+        // extract_tests!(
+        //     uint64,
+        //     item: 245361314_u64,
+        //     fill_expected_buf: |buf: &mut BytesMut, val: u64| encode_varint(val, buf)
+        // );
+        // extract_tests!(
+        //     sint32,
+        //     item: -21314_i32,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| encode_varint(((val << 1) ^ (val >> 31)) as u32 as u64, buf)
+        // );
+        // extract_tests!(
+        //     sint64,
+        //     item: -245361314_i64,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| encode_varint(((val << 1) ^ (val >> 63)) as u64, buf)
+        // );
+        //
+        // // Test single 32/64 const size type
+        // extract_tests!(
+        //     float,
+        //     item: 4543.342_f32,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| buf.put_f32_le(val)
+        // );
+        // extract_tests!(
+        //     double,
+        //     item: 4543986.342542_f64,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| buf.put_f64_le(val)
+        // );
+        // extract_tests!(
+        //     fixed32,
+        //     item: 4543_u32,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| buf.put_u32_le(val)
+        // );
+        // extract_tests!(
+        //     fixed64,
+        //     item: 349320_u64,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| buf.put_u64_le(val)
+        // );
+        // extract_tests!(
+        //     sfixed32,
+        //     item: -4543_i32,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| buf.put_i32_le(val)
+        // );
+        // extract_tests!(
+        //     sfixed64,
+        //     item: -349320_i64,
+        //     fill_expected_buf: |buf: &mut BytesMut, val| buf.put_i64_le(val)
+        // );
 
         // Test single length delimited type
         extract_tests!(
             string,
             item: "my awesome string".to_string(),
             fill_expected_buf: |buf: &mut BytesMut, val: String| {
-                encode_varint(val.len().try_into().unwrap(), buf);
+                //encode_varint(val.len().try_into().unwrap(), buf);
```

Comment:
Should we remove this? We have the PR which shows what was removed if we wanna go back to the previous state.